picard/webservice/throttle.py

```diff
@@ -128,12 +128,15 @@ def decr_requests(hostkey):
               hostkey, CONGESTION_UNACK[hostkey])
 
 
-def copy_minimal_delay(original_host_key, redirect_host_key):
-    if (original_host_key in REQUEST_DELAY_MINIMUM
-            and redirect_host_key not in REQUEST_DELAY_MINIMUM):
-        log.debug("throttle: Setting the minimum rate limit for %s to %i" %
-                  (redirect_host_key, REQUEST_DELAY_MINIMUM[original_host_key]))
-        REQUEST_DELAY_MINIMUM[redirect_host_key] = REQUEST_DELAY_MINIMUM[original_host_key]
+def copy_minimal_delay(from_hostkey, to_hostkey):
+    """Copy minimal delay from one hostkey to another
+        Useful for redirections
+    """
+    if (from_hostkey in REQUEST_DELAY_MINIMUM
+            and to_hostkey not in REQUEST_DELAY_MINIMUM):
+        log.debug("throttle: %s: Setting the minimum rate limit to %i" %
```

Comment:
We can avoid % when logging and simply pass the values as parameters.

Reply:
> We can avoid % when logging and simply pass the values as parameters.

You're totally right, i'll fix that.